src/DateTimeInput/index.jsx

```diff
@@ -166,6 +166,10 @@ DateTimeInput.propTypes =
      *  Display as error/invalid
      */
     hasError              : PropTypes.bool,
+    /**
+     *  Display TimeInput if true
+     */
+    hasTimeInput          : PropTypes.bool,
```

Comment:
Just need to remove `hasTimeInput` from `DateTimeInput.propTypes` and we’re good to merge